src/main/java/com/shopify/model/fulfillmentOrderApi/ShopifyDestination.java

```diff
@@ -0,0 +1,122 @@
+package com.shopify.model.fulfillmentOrderApi;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ShopifyDestination {
+
+	private String id;
+	private String address1;
+	private String address2;
+	private String city;
+	private String company;
+	private String country;
+	private String email;
+	@XmlElement(name = "first_name")
+	private String firstName;
+	@XmlElement(name = "last_name")
+	private String lastname;
```

Comment:
lastName should have capital n in name. It shouldn't matter because of your annotation but better to fix just in case there is some weird marshaling thing.